src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -297,6 +316,27 @@ public long StartBackupTask(long taskId, bool isFullBackup)
             return CreateBackupTask(periodicBackup, isFullBackup);
         }
 
+        public long GetWakeDatabaseTime()
+        {
+            var wakeupDatabase = long.MaxValue;
+            long lastEtag;
+
+            using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+            using (var tx = context.OpenReadTransaction())
+            {
+                lastEtag = DocumentsStorage.ReadLastEtag(tx.InnerTransaction);
+            }
+
+            foreach (var backup in _periodicBackups)
+            {
+                var nextBackup = GetNextWakeupTime(lastEtag, backup.Value.Configuration, backup.Value.BackupStatus);
+                if (nextBackup == null)
+                    continue;
+                wakeupDatabase = Math.Min(wakeupDatabase, nextBackup.Value.Ticks);
+            }
+            return wakeupDatabase;
```

Comment:
Note, you are returning here time in ticks, need to make sure that you don't try to make that into the ticks to wait for wakeup